apps/wallet/src/ui/app/pages/initialize/select/index.tsx

```diff
@@ -28,17 +30,38 @@ const selections = [
 const SelectPage = () => {
     return (
         <>
-            <h1 className={st.headerTitle}>New to Sui Wallet?</h1>
-            <div className={st.selector}>
+            <Heading variant="heading1" color="gray-90" as="h2" weight="bold">
+                New to Sui Wallet?
+            </Heading>
+            <div className="flex flex-col flex-nowrap gap-7.5 mt-7">
                 {selections.map((aSelection) => (
-                    <div className={st.card} key={aSelection.url}>
-                        <h3 className={st.title}>{aSelection.title}</h3>
-                        <div className={st.desc}>{aSelection.desc}</div>
+                    <div
+                        className={cl(
+                            st.card,
```

Comment:
If you're doing a partial tailwind migration why not do it all?

Reply:
done